### DISCUSSION TITLE
OpenGL errors when running osgearth_windows.exe application...

### DISCUSSION THREAD

**Discussion by keince** [pelicanmapping/osgearth#1816] [Support Q&A] [answered]
When I run the osgearth_windows.exe application in the osgEarth build, I'm seeing the following OpenGL errors when I create and delete Views...

Warning: detected OpenGL error 'invalid operation' at start of State::apply()
Warning: detected OpenGL error 'invalid operation' after applying GLMode 0xde1
Warning: detected OpenGL error 'invalid operation' after applying attribute Texture2D 0000021A6978AC00
Warning: detected OpenGL error 'invalid operation' after applying GLMode 0xb10
Warning: detected OpenGL error 'invalid operation' after applying GLMode 0xb24
Warning: detected OpenGL error 'invalid operation' after applying GLMode 0xb50
Warning: detected OpenGL error 'invalid operation' after applying GLMode 0xb71
Warning: detected OpenGL error 'invalid operation' after applying GLMode 0x4000
Warning: detected OpenGL error 'invalid operation' after applying attribute MaterialGL3 0000021A68438C50
Warning: detected OpenGL error 'invalid operation' after applying attribute Point 0000021A68C10D10
Warning: detected OpenGL error 'invalid operation' after applying attribute LineWidth 0000021A6871BD90
Warning: detected OpenGL error 'invalid operation' after applying attribute LineStipple 0000021A6871CB10
Warning: detected OpenGL error 'invalid operation' after applying attribute LightModel 0000021A68C10090
Warning: detected OpenGL error 'invalid operation' after applying attribute BlendFunc 0000021A6871BBE0
Warning: detected OpenGL error 'invalid operation' after applying attribute ColorMask 0000021A6871C7B0
Warning: detected OpenGL error 'invalid operation' after applying attribute Viewport 0000021A68C11850
Warning: detected OpenGL error 'invalid operation' after applying attribute VirtualProgram 0000021A68099800
Warning: detected OpenGL error 'invalid operation' at end of State::apply()
Warning: detected OpenGL error 'invalid operation' at end of SceneView::draw()

Is there something I can do to alleviate these errors? I bring this up because we're seeing the same errors in our osgEarth application in multiwindow mode using CompositeViewer when we create/delete windows.

We're using osgEarth 3.1 on Windows x64

Thanks for any help in advance...

**Comment by gwaldron** [comment, marked as answer]
This is a known issue - currently unresolved. I am going to convert this into an Issue for tracking. Thanks.

**Reply by keince** [reply]
Okay. Thanks Glenn.
